src/Model/APContact.php

```diff
@@ -304,6 +304,10 @@ public static function getByURL($url, $update = null)
 			$apcontact['baseurl'] = null;
 		}
 
+		if (empty($apcontact['subscribe'])) {
+			$apcontact['subscribe'] = null;
```

Comment:
Why not creating the array index on line 137 when the array is declared?

Reply:
I just replicated the line above for a quick fix. Today I haven't got time for more.